microprofile/server/src/main/java/io/helidon/microprofile/server/Server.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018-2020 Oracle and/or its affiliates.
```

Comment:
Copyright years should be comma separated in Helido.

Reply:
Hmm... I'm pretty sure I've seen dash used somewhere, so we are definitely not consistent.

Reply:
Fixed.